minions/async/src/main/java/io/github/microcks/minion/async/producer/ProducerManager.java

```diff
@@ -37,17 +36,19 @@
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
+import java.net.URLEncoder;
+import java.nio.charset.Charset;
 import java.util.Arrays;
-import java.util.HashSet;
 import java.util.Set;
-import java.util.stream.Collectors;
 
 @Unremovable
 @ApplicationScoped
 /**
- * ProducerManager is the responsible for emitting mock event messages when specific frequency triggered is reached.
- * Need to specify it as @Unremovable to avoid Quarkus ARC optimization removing beans that are not injected elsewhere
- * (this one is resolved using Arc.container().instance() method from ProducerScheduler).
+ * ProducerManager is the responsible for emitting mock event messages when specific frequency
+ * triggered is reached. Need to specify it as @Unremovable to avoid Quarkus ARC optimization
+ * removing beans that are not injected elsewhere (this one is resolved using
+ * Arc.container().instance() method from ProducerScheduler).
+ * 
```

Comment:
Could you please keep previous formatting to avoid uncessary changes in commit? Thanks!